app/checkout/checkout.jsx

```diff
@@ -1,10 +1,35 @@
 import { Header } from "../header";
+import { useLocation } from "react-router";
+import { useState } from "react";
```

Comment:
the other file uses single quotes, this file uses double quotes. just gotta pick one and make it consistent. i prefer single quotes for this as it's more js-like code writing.